docs/_posts/murat-gunay/2021-12-10-classifierdl_bert_news_ur.md

```diff
@@ -0,0 +1,118 @@
+---
+layout: model
+title: News Classifier for Urdu text
+author: John Snow Labs
+name: classifierdl_bert_news
+date: 2021-12-10
+tags: [urdu, news, classifier, ur, open_source]
+task: Text Classification
+language: ur
+edition: Spark NLP 3.3.0
+spark_version: 2.4
+supported: true
+article_header:
+  type: cover
+use_language_switcher: "Python-Scala-Java"
+---
+
+## Description
+
+Classify Urdu news into 7 categories.
+
+## Predicted Entities
+
+`business`, `entertainment`, `health`, `inland`, `science`, `sports`, `weird_news`
+
+{:.btn-box}
+<button class="button button-orange" disabled>Live Demo</button>
+<button class="button button-orange" disabled>Open in Colab</button>
+[Download](https://s3.amazonaws.com/auxdata.johnsnowlabs.com/public/models/classifierdl_bert_news_ur_3.3.0_2.4_1639125233132.zip){:.button.button-orange.button-orange-trans.arr.button-icon}
+
+## How to use
+
+
+
+<div class="tabs-box" markdown="1">
+{% include programmingLanguageSelectScalaPythonNLU.html %}
+```python
+document_assembler = DocumentAssembler() \
+      .setInputCol("news") \
+      .setOutputCol("document")
+
+embeddings = BertSentenceEmbeddings.pretrained("labse", "xx") \
+      .setInputCols("document") \
+      .setOutputCol("sentence_embeddings")
+
+classifierdl = ClassifierDLModel.pretrained("classifierdl_bert_news", "ur") \
+  .setInputCols(["document", "sentence_embeddings"]) \
+  .setOutputCol("class")
+
+urdu_news_pipeline = Pipeline(stages=[document_assembler, embeddings, classifierdl])
+light_pipeline = LightPipeline(urdu_news_pipeline.fit(spark.createDataFrame([['']]).toDF("news")))
+
+result = light_pipeline.annotate(" سینٹ جیسی فاؤنڈيشنز کے مصنوعی ذہانت کے اداروں میں بہت سے امريکی ماہرين دلچسپ کام کر رہے ہیں۔ ای بے کے پاس مصنوعی ذہانت کے 45 مختلف شعبے ہیں اور اس نے مشرقی مصنوعی ذہانت کی کمپنیاں خریدنے کے لیے پانچ ارب پاؤنڈز خرچ کیے ہیں۔'")
+result["class"]
+```
+```scala
+val document = DocumentAssembler()
+    .setInputCol("text")
+    .setOutputCol("document")
+
+val embeddings = BertSentenceEmbeddings
+    .pretrained("lanse", "xx") 
+    .setInputCols("document")
+    .setOutputCol("sentence_embeddings")
+
+val document_classifier = ClassifierDLModel.pretrained("classifierdl_bert_news", "ur") 
+  .setInputCols(Array("document", "sentence_embeddings")) 
+  .setOutputCol("class")
+
+val nlpPipeline = new Pipeline().setStages(Array(document, embeddings, document_classifier))
+val light_pipeline = LightPipeline(nlpPipeline.fit(spark.createDataFrame([[""]]).toDF("text")))
+val result = light_pipeline.annotate(" سینٹ جیسی فاؤنڈيشنز کے مصنوعی ذہانت کے اداروں میں بہت سے امريکی ماہرين دلچسپ کام کر رہے ہیں۔ ای بے کے پاس مصنوعی ذہانت کے 45 مختلف شعبے ہیں اور اس نے مشرقی مصنوعی ذہانت کی کمپنیاں خریدنے کے لیے پانچ ارب پاؤنڈز خرچ کیے ہیں۔'")
+
+```
+</div>
+
+## Results
+
+```bash
+['weird_news']
```

Comment:
It's a science or business news, not a `weird_news`. Can you pick another example?

![image](https://user-images.githubusercontent.com/36634572/145552213-a27cf832-65d8-41f7-9f64-f9f7971b77a8.png)